ci-operator/step-registry/openshift-logging/enable-cluster-logging/openshift-logging-enable-cluster-logging-ref.yaml

```diff
@@ -0,0 +1,43 @@
+ref:
+  as: openshift-logging-enable-cluster-logging
+  from: tests-private
+  grace_period: 10m
+  commands: openshift-logging-enable-cluster-logging-commands.sh
+  cli: latest
+  timeout: 12h0m0s
+  env:
+  - name: CLF_NAME
+    default: "collector"
+    documentation: The operator channel to be deployed
+  - name: CLF_NAMESPACE
+    default: "openshift-logging"
+    documentation: The namespace where clf is deployed
+  - name: LOKI_NAME
+    default: "logging-loki"
+    documentation: The operator channel to be deployed
+  - name: LOKI_NAMESPACE
+    default: "openshift-logging"
+    documentation: The namespace where lokistack is deployed
+  - name: MINIO_NAMESPACE
+    default: "openshift-logging-minio"
+    documentation: The namespace for minio
+  - name: MINIO_BUCKET
+    default: "logging-loki"
+    documentation: the bucket name for lokistack
+  - name: SIZE
+    default: "1x.demo"
+    documentation: The operator channel to be deployed
+  - name: INPUTS
```

Comment:
This env var is not used. Must we add it?

Reply:
Add code to set inputRef from INPUTS.